cryostat/README.md

```diff
@@ -32,15 +32,14 @@ A Helm chart for deploying [Cryostat](https://cryostat.io/) on Kubernetes and Op
 | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------- |
 | `grafana`                                         | Configuration for the customized Grafana instance for Cryostat                                                                                                                                                                                                                                                |                                               |
 | `grafana.image.repository`                        | Repository for the Grafana container image                                                                                                                                                                                                                                                                    | `quay.io/cryostat/cryostat-grafana-dashboard` |
-| `grafana.image.pullPolicy`                        | Image pull policy for the Grafana container image                                                                                                                                                                                                                                                             | `Always`                                      |
-| `grafana.image.tag`                               | Tag for the Grafana container image                                                                                                                                                                                                                                                                           | `latest`                                      |
+| `grafana.image.pullPolicy`                        | Image pull policy for the Grafana container image                                                                                                                                                                                                                                                             | `IfNotPresent`                                |
+| `grafana.image.tag`                               | Tag for the Grafana container image                                                                                                                                                                                                                                                                           | `cryostat-v2.3.0`                             |
```

Comment:
This should be `2.3.0` shouldn't it?

Reply:
Both tags exist pointing to the same image. I can set it to `2.3.0`